### PR TITLE
feat: Add an employee scheduling Python quickstart

### PR DESCRIPTION
**Pull request by Christopher-Chianelli**
## Description of the change

Add an employee scheduling quickstart for Python.

### Development

- [ ] The changes have been covered with tests, if necessary.
- [ ] You have a green build, with the exception of the flaky tests.
- [ ] UI and JS files are fully tested, the user interface works for all modules affected by your changes (e.g., solve and analyze buttons).
- [ ] The network calls work for all modules affected by your changes (e.g., solving a problem).
- [ ] The console messages are validated for all modules affected by your changes.

### Code Review

- [ ] This pull request includes an explanatory title and description.
- [ ] The GitHub issue is linked.
- [ ] At least one other engineer has approved the changes.
- [ ] After PR is merged, inform the reporter.

### CODE REVIEW

File: python/employee-scheduling/src/employee_scheduling/routes.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+app = FastAPI(docs_url='/q/swagger-ui')
+data_sets: dict[str, EmployeeSchedule] = {}
+
+
+@app.get("/demo-data")
+async def demo_data_list() -> list[DemoData]:
+    return [e for e in DemoData]
+
+
+@app.get("/demo-data/{dataset_id}",  response_model_exclude_none=True)
```
Comment (zepfred) [inline python/employee-scheduling/src/employee_scheduling/routes.py:35]: It looks like we always return the `SMALL` dataset

**Review by zepfred — COMMENTED** [review]
The logic looks good, but I couldn't run it locally. I want to make sure it works locally before approving the PR.

**Comment by Christopher-Chianelli** [comment]
It required https://github.com/TimefoldAI/timefold-solver-python/pull/68 (which was only merged into main just now)

File: python/employee-scheduling/src/employee_scheduling/routes.py (hunk 1 above)
Comment (Christopher-Chianelli) [reply python/employee-scheduling/src/employee_scheduling/routes.py:35]: Well, there is no LARGE dataset defined for the employee scheduling Java quickstart; note `generateDemoData` takes no arguments: https://github.com/TimefoldAI/timefold-quickstarts/blob/0edf2339ca75726f068153b4b496385f6b7d7be1/use-cases/employee-scheduling/src/main/java/org/acme/employeescheduling/rest/DemoDataGenerator.java#L55 ; the `DemoData` is only there for ceremony (namely, making it work with the shared logic which expects an enum); `SMALL == LARGE`.

File: python/employee-scheduling/static/index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        <p>Generate the optimal schedule for your employees.</p>
+
+        <div class="mb-4">
+            <button id="solveButton" type="button" class="btn btn-success">
+                <span class="fas fa-play"></span> Solve
+            </button>
+            <button id="stopSolvingButton" type="button" class="btn btn-danger">
+                <span class="fas fa-stop"></span> Stop solving
+            </button>
+            <button id="publish" type="button" class="btn btn-primary">
```
Comment (zepfred) [inline python/employee-scheduling/static/index.html]: This button is not used. Let's either remove it or add the necessary logic for publishing schedules.

**Review by zepfred — APPROVED** [review]
Looks good! +1 after addressing the comment regarding the publish button.

File: python/employee-scheduling/static/index.html (hunk 2 above)
Comment (triceo) [reply python/employee-scheduling/static/index.html]: As far as I know, the button is also not present in the Java counterpart to this quickstart. (At the very least it was discussed.) So let's remove altogether, incl. anything else (class fields etc.) that perhaps references the publishing functionality.

**Pull request merged**